config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -470,3 +470,49 @@ presubmits:
       testgrid-alert-stale-results-hours: '24'
       testgrid-create-test-group: 'true'
       testgrid-dashboards: sig-testing-kind
+
+  - name: pull-kubernetes-conformance-kind-ga-only-parallel
+    optional: true
+    always_run: true
+    decorate: true
+    skip_branches:
```

Comment:
Trying to decide whether to scope this with a run_if_changed, at least initially. There's not a super-clear subset of things we could limit this to

Reply:
run_if_changed is hard to get right, I generally don't recommend it.